component/src/main/java/com/celements/blog/service/BlogService.java

```diff
@@ -176,10 +172,10 @@ public boolean isSubscribable(DocumentReference blogConfDocRef) throws XWikiExce
     LOGGER.debug("getSubribedToBlogs: for blog '" + blogConfDocRef + "' got:" + ret);
     return Collections.unmodifiableList(ret);
   }
-  
+
   @Override
-  public List<SpaceReference> getSubribedToBlogsSpaceRefs(DocumentReference blogConfDocRef
-      ) throws QueryException, XWikiException {
+  public List<SpaceReference> getSubribedToBlogsSpaceRefs(DocumentReference blogConfDocRef)
+      throws QueryException, XWikiException {
     List<SpaceReference> ret = new ArrayList<SpaceReference>();
```

Comment:
my Clean up removes redundant type arguments

> Java > Code Style > Clean up | Unnecessary Code | Remove redundant type arguments (1.7 or higher)

This doesn't seem to be activated or available in your workspace, perhaps you aren't using Eclipse Neon (4.6+) or Java 7.